src/components/type-group/type-option/TypeOption.js

```diff
@@ -18,7 +21,8 @@ class TypeOption extends Component {
                         checked={ selected }
                         disabled={ !selectable }
                         onChange={ this.handleInputChange }
-                        onClick={ this.handleClick } />
+                        onClick={ this.handleClick }
```

Comment:
I think we can remove the `onClick`.

Reply:
You're right. I'm already spreading the input props.